src/lib/i18n/locales/es-ES/translation.json

```diff
@@ -14,61 +14,61 @@
 	"*Prompt node ID(s) are required for image generation": "Los ID de nodo son requeridos para la generación de imágenes",
 	"A new version (v{{LATEST_VERSION}}) is now available.": "Nueva versión (v{{LATEST_VERSION}}) disponible.",
 	"A task model is used when performing tasks such as generating titles for chats and web search queries": "El modelo de tareas realiza tareas como la generación de títulos para chats y consultas de búsqueda web",
-	"a user": "un/a usuari@",
+	"a user": "un usuario",
 	"About": "Acerca de",
 	"Accept autocomplete generation / Jump to prompt variable": "Aceptar generación de autocompletado / Saltar a indicador variable",
 	"Access": "Acceso",
 	"Access Control": "Control de Acceso",
-	"Accessible to all users": "Accesible para todos l@s usuari@s",
+	"Accessible to all users": "Accesible para todos los usuarios",
 	"Account": "Cuenta",
 	"Account Activation Pending": "Activación de cuenta Pendiente",
 	"Accurate information": "Información precisa",
 	"Actions": "Acciones",
 	"Activate": "Activar",
 	"Activate this command by typing \"/{{COMMAND}}\" to chat input.": "Activar este comando escribiendo \"/{{COMMAND}}\" en el chat",
-	"Active Users": "Usuari@s activos",
+	"Active Users": "Usuarios activos",
 	"Add": "Añadir",
 	"Add a model ID": "Añadir un ID de modelo",
 	"Add a short description about what this model does": "Añadir una breve descripción sobre lo que hace este modelo",
 	"Add a tag": "Añadir una etiqueta",
-	"Add Arena Model": "Añadir un modelo a la Arena",
+	"Add Arena Model": "Añadir modelo Arena",
```

Comment:
This one is wrong, it should be "a la arena"

Reply:
Updated, thanks for checking